### ISSUE TITLE
Logistical sorter problem

### ISSUE THREAD

**Issue by johndu132** [mekanism/Mekanism#7590]
### Issue description

It seems with the logistical sorter it ignores the filter list.  I have a set up that has two coming from a single chest, one that filters rotten flesh and the other filters golden nuggets.  However regardless of which logistical sorter has what filters, both items go through the sorters.  There also seems to be a bug where these stop working when the save is reloaded.

### Minecraft version

1.19.1 or 1.19.2 (Latest)

### Forge version

43.1.2

### Mekanism version

10.3.2 (Latest)

**Comment by beholderface** [comment]
Do you have them set to Allow Default?

**Comment by johndu132** [comment]
Do you mean having the back in the lower left where the colors are set to none?  Yeah, my sorters are all set to none.  The only filter on is the ItemStack.

**Comment by pupnewfster** [comment]
![image](https://user-images.githubusercontent.com/1203288/186510575-51effef1-320e-4076-8c9d-be966862a1fa.png)
that thing. If you have it on it uses the default color/settings for everything that doesn't match a filter.

**Comment by johndu132** [comment]
I have that turned off as well since I didn't think it would be needed because of me only needing to sort two items.  I think I may have found my fix though.  There must have been a mix-up in the Mekanism config folder.  I deleted it, reloaded my game, and now the sorters are working properly and they're not stopping when I reload my save.  This was a bit strange, but I did this since I was updating from the 1.16.5 version of Minecraft and it made me think that the new Mekanism didn't overwrite the previous version's config folder.  It seems to have worked for now.
